contracts/src/v0.7/interfaces/ArbitrumInboxInterface.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity ^0.7.0;
+
+interface IBridge {
```

Comment:
Can we break these other interfaces out into their own files? Also, let's make the file names match the interface name. I think both are requirements to work with Truffle.

Reply:
Let's stick with the existing `*Interface` naming pattern too. Would be happy to change it in the future though ;)